http/http-advanced-reactive/src/main/java/io/quarkus/ts/http/advanced/reactive/HeadersResource.java

```diff
@@ -28,4 +30,12 @@ public Uni<Response> headersOverride() {
         return Uni.createFrom().item(response);
     }
 
+    @GET
+    @Path("/no-accept")
+    @Produces(MediaType.TEXT_PLAIN)
+    public Uni<Response> okHeaders() {
+        final Response response = Response.ok("ok headers").build();
+        return Uni.createFrom().item(response);
```

Comment:
Afaik wouldn't be enough to send return just `Uni.createFrom().item("ok headers");` ? 
I don't say this is wrong! Just in regard to other endpoints only `headersOverride` using it and in addition to string  it's setting header.